project_timeline/tests/__init__.py

```diff
@@ -1 +1 @@
-from . import test_project_timeline
+from . import test_demo, test_project_timeline
```

Comment:
This should be this way:

```suggestion
from . import test_demo
from . import test_project_timeline
```

But anyway, don't test demo data. That's not sane nor resilient: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#avoid-relying-on-demo-data

Reply:
Ok I'm used to this in my projects but it makes sense for OCA to play well with other modules

In any case this will be checked by the constraint in https://github.com/OCA/project/pull/1260